include/nlohmann/detail/exceptions.hpp

```diff
@@ -131,16 +131,30 @@ class exception : public std::exception
         {
             return concat(a, '/', detail::escape(b));
         });
-        return concat('(', str, ") ");
+
+        return concat('(', str, ") ") + get_byte_positions(leaf_element);
```

Comment:
Nit: you can also include the `get_byte_positions...` in the `concat` call instead of concatenating with `+` operator. i.e. `concat('(', str, ") ", get_byte_positions(leaf_element))`

Reply:
Aside from readability, I believe using `concat` also has a performance benefit.